src/Model/Command/UserRecommendation.php

```diff
@@ -339,4 +339,4 @@ protected function getCommandParameters(): array
 
         return $parameters;
     }
-}
+}
```

Comment:
Missing newline at the end of file:

```suggestion
}

```